wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -305,7 +321,7 @@ async def _inject_defaults(
         namespace: str,
         core_api: "CoreV1Api",
     ) -> Tuple[Dict[str, Any], Optional["V1Secret"]]:
-        """Apply our default values, return job dict and secret.
+        """Apply our default values, return job dict and api key secret.
```

Comment:
We weren't really doing anything with the imagepull secret returned from this, so I repurposed it and now it returns the API key secret

Reply:
this seems really suspicious but you're totally right.  wasn't even used when it was introduced, afaict